Refactor based on new website. (#3)
veecle/.github: profile/README.md

# Veecle

> Our mission is to bridge the gap between existing supplier components and smart apps running on top through unified software framework. We provide a platform to tackle next-gen automotive connected & security demands.

Reviewer: ```suggestion
Our mission is to bridge the gap between existing supplier components and smart apps running on top through a unified software framework. We provide a platform to tackle next-gen automotive connected & security demands.
```

Veecle contributes with two keys components to the ecosystem:

- [Veecle-pxros](https://github.com/veecle/veecle-pxros) A runtime for writing Rust applications on top of HighTec's safety kernel for the Infineon AURIX™ Platform.
- [Tricore-probe](https://github.com/veecle/tricore-probe): A tool that facilitates the flashing and debugging of Rust code on AURIX™ chips.

### Android Automotive support

We support [Android Automotive](https://source.android.com/docs/automotive/start/what_automotive): this enables customers to develop safe and efficient interfaces to the vehicle. 

This is not public (yet). You can find information [here](https://veecle.io/tech/aaos).

### Robot Operating System support

We integrate to the [Robot Operating System](https://www.ros.org) to allow for easy integration with enabled devices over the [Data Distribution Service](https://www.dds-foundation.org/what-is-dds-3). 

This is not public (yet). You can find information [here](https://veecle.io/tech/dds).

## About Veecle

> Veecle sits between a legacy electrical/electronic (E/E) architecture, and modern UI and Cloud frameworks. Veecle abstracts an E/E architecture and provides unified control over it using a modern API. Veecle allows companies to develop applications using unified APIs to interact with in-vehicle functions. An optional front-end framework is provided through Android Automotive support.

Reviewer: ```suggestion
Veecle
* sits between a legacy electrical/electronic (E/E) architecture, and modern UI and Cloud frameworks.
* abstracts an E/E architecture and provides unified control over it using a modern API.
* allows companies to develop applications using unified APIs to interact with in-vehicle functions. An optional front-end framework is provided through Android Automotive support.
```

Its a lot of Veecle does.

Reviewer: It's fine but not really a nice paragraph structure. If possible, this should be changed on the website too.

> Our runtime seamlessly integrates across many ECUs — from infotainment systems to real-time controllers, playing together with existing and external services. The very same Rust code base can be compiled for Linux (e.g. Android Automotive) or as an isolated task on a real-time operating system (RTOS), ensuring *freedom from interference* on e.g. Infineon AURIX™ chips, or ARM embedded controllers.

Reviewer: ```suggestion
Our runtime seamlessly integrates across many ECUs — from infotainment systems to real-time controllers, cooperating with existing and external services. The very same Rust code base can be compiled for Linux (e.g. Android Automotive) or as an isolated task on a real-time operating system (RTOS), ensuring *freedom from interference* on e.g. Infineon AURIX™ chips, or ARM embedded controllers.
```

# Veecle

Our mission is to bridge the gap between existing supplier components and smart apps running on top through unified software framework. We provide a platform to tackle next-gen automotive connected & security demands.

### AURIX™ Rust Startup Ecosystem

The AURIX™ Rust Startup Ecosystem is a collaborative effort involving [Veecle](https://www.veecle.io), [Infineon](https://www.infineon.com), [HighTec](https://hightec-rt.com) and [Bluewind](https://www.bluewind.it) aimed at supporting Rust on Infineon's AURIX™ architecture for automotive and industrial applications.

Veecle contributes with two keys components to the ecosystem:

- [Veecle-pxros](https://github.com/veecle/veecle-pxros) A runtime for writing Rust applications on top of HighTec's safety kernel for the Infineon AURIX™ Platform.
- [Tricore-probe](https://github.com/veecle/tricore-probe): A tool that facilitates the flashing and debugging of Rust code on AURIX™ chips.

### Android Automotive support

> We support [Android Automotive](aaos): this enables customers to develop safe and efficient interfaces to the vehicle. 

Reviewer 2: The link label is not being resolved if I view the file here:

https://github.com/veecle/.github/blob/b7831ef6e59043330de3d79e6e1b20a70b70b3ef/profile/README.md

I think this happens with the rest of links.

This is not public (yet). You can find information [here](aaos-plugin).

### Robot Operating System support

We integrate to the [Robot Operating System](ros) to allow for easy integration with enabled devices over the [Data Distribution Service](dds). 

This is not public (yet). You can find information [here](ros-plugin).

## About Veecle

> Veecle sits between a legacy E/E architecture and modern UI and Cloud frameworks. Veecle abstracts an E/E architecture and provides unified control over it using a modern API. Veecle allows companies to develop "Apps" using unified APIs to interact with in-vehicle functions. An optional front-end framework is provided through Android Automotive support.

Reviewer 2: I do not know what E/E means, but perhaps I'm just not familiar with the industry. Perhaps it should be expanded.

As I have already opened a comment:

```suggestion
Veecle sits between a legacy E/E architecture, and modern UI and Cloud frameworks. Veecle abstracts an E/E architecture and provides unified control over it using a modern API. Veecle allows companies to develop applications using unified APIs to interact with in-vehicle functions. An optional front-end framework is provided through Android Automotive support.
```

(I don't think capitalization nor quotes of "apps" is needed. I prefer "application", but that's a personal taste.)
